PLUGINS.md

```diff
@@ -284,6 +284,14 @@ def generate_init_code(self, generated_code: str) -> str:
 
 Hook executed on generation of init code. Result is used as content of `__init__.py`.
 
+### process_name
+
+```py
+def process_name(self, name: str) -> str:
```

Comment:
Can we provide more context about the item which name is being processed by this hook, eg. an AST node? This would let plugin authors use different logic for different types.

Reply:
Done